src/common/src/config.rs

```diff
@@ -424,10 +424,6 @@ pub struct BatchConfig {
 /// The section `[streaming]` in `risingwave.toml`.
 #[derive(Clone, Debug, Serialize, Deserialize, DefaultFromSerde)]
 pub struct StreamingConfig {
-    /// The maximum number of barriers in-flight in the compute nodes.
-    #[serde(default = "default::streaming::in_flight_barrier_nums")]
-    pub in_flight_barrier_nums: usize,
```

Comment:
Is this a breaking change?

Reply:
https://github.com/risingwavelabs/risingwave/blob/5c588ef352cf9710481b439305f54af81d1ee479/src/common/src/config.rs#L67-L85

Seems not

Reply:
But https://github.com/risingwavelabs/risingwave/pull/9156 also mentioned that

> We may change the behavior to return an error if necessary in the future, to avoid https://github.com/risingwavelabs/risingwave/issues/9153 from happening as much as possible.

🤔

Reply:
If we want to be extra safe, may try to add a test case like this https://github.com/risingwavelabs/risingwave/pull/13824 🤪